pkg/apis/deviceplugin/v1/gpudeviceplugin_types.go

```diff
@@ -31,11 +31,12 @@ type GpuDevicePluginSpec struct {
 	// Image is a container image with GPU device plugin executable.
 	Image string `json:"image,omitempty"`
 
-	// InitImage is a container image with tools (e.g., GPU NFD source hook) installed on each node.
+	// Deprecated: InitImage is a container image with tools (e.g., GPU NFD source hook) installed on each node.
```

Comment:
From #1533: can we come up with other use for the initcontainer? For example, QAT initcontainer is there to enable SR-IOV VFs which takes it away from cluster-admins.

Reply:
It is theoretically possible that some other use would be made into the gpu-initcontainer. In that sense, maybe deprecating it is not wise.

Though, DRA has/will have built-in support for SR-IOV.

Reply:
Perhaps not deprecate GPU just yet? It could be handly for static configuration

Reply:
yep, I'll remove deprecation from GPU's initImage.